machine/machine_core/machine_virtual.py

```diff
@@ -687,5 +687,5 @@ def add_netiface(self, networking=None):
 
     def needs_writable_usr(self):
         # On atomic systems, we need a hack to change files in /usr/lib/systemd
-        if self.atomic_image:
+        if self.atomic_image or "coreos" in self.image:
```

Comment:
this could just be `if self.ostree_image:`

Reply:
Indeed, and I had that locally, but forgot to `--amend` it, bah! Pushed.